server/src/services/findSeason.js

```diff
@@ -0,0 +1,15 @@
+import { SeasonAverage } from "../models/index.js"
+
+const findSeason = async (player, table) => {
```

Comment:
maybe update the name to `findSeasonAndRelateToTable` so it's more descriptive when used in your router since it doesn't return anything